features/support/env.rb

```diff
@@ -52,9 +52,14 @@
 #     DatabaseCleaner.strategy = :transaction
 #   end
 #
-
+options = {
+  :window_size => [6000, 3000]
+}
```

Comment:
Already included in previous PR, but to pass CI, we have to resize window for webdriver

Please ignore this file.